internal/pkg/cli/svc_delete.go

```diff
@@ -99,11 +99,6 @@ func newDeleteSvcOpts(vars deleteSvcVars) (*deleteSvcOpts, error) {
 
 // Validate returns an error if the user inputs are invalid.
 func (o *deleteSvcOpts) Validate() error {
-	if o.name != "" {
-		if _, err := o.store.GetService(o.appName, o.name); err != nil {
-			return err
-		}
-	}
 	if o.envName != "" {
 		return o.validateEnvName()
```

Comment:
should we just have return `nil` for `Validate`?
this could also blow up if the app name is empty